orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -5334,3 +5335,12 @@ def run_step(model, x):
     _test_helpers.assert_values_are_close(pt_prediction, ort_prediction)
     _test_helpers.assert_values_are_close(pt_loss, ort_loss)
     _test_helpers.assert_values_are_close(pt_x.grad, ort_x.grad)
+
+
+def test_eval_model_mode():
+    device = "cuda"
+    d_in, h_size, d_out = 2, 2, 2
+    model = NeuralNetSinglePositionalArgument(d_in, h_size, d_out).to(device)
+    model.eval()
+    ort_model = ORTModule(model)
+    assert ort_model.training is False
```

Comment:
```suggestion
    assert ort_model.training is False
    assert ort_model._torch_module.training is False
```
Just to make sure the ORTModule instance (user facing torch.nn.Module) matches the internal one (original module before wrapping)